### ISSUE TITLE
Feature Request: Allow alternative sorting of files

### ISSUE THREAD

**Issue by h3nn3s** [beechit/fal_securedownload#135]
Files are corrently only sorted by filename (FAL default).
For frontend purposes is could be nice to use different fields for sorting.

Scenario:
You output a bunch of files and show the file title when given. Now your list is not sorted alphabetically anymore, because first char of title and filename must not be the same.

I already made some research but did not find the right place to implement something. Currently I cannot provide a PR.

kind regards
henrik

**Comment by gilas1959** [comment]
I agree, unfortunately I have a page with several files to download and it is a mess! I hope the author can add some fields within the content plugin, such as in news (sort type: creation time, modify time,  alphabetic, sort direction: ascending, descending)

another request: printout only the records selected with the plugin, associated one or more category (ie. I have a file that is related to the category "documentation", I put the plugin on the page -> if I don't select any category, all files will be printed out form the folder, otherwise if I configure the plugin with the category "documentation" will be printed only the file related to this category)


thank you very much
